pkg/cmd/kpt/update/update.go

```diff
@@ -94,6 +97,7 @@ func (o *Options) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVarP(&o.RepositoryName, "repo", "r", "", "filter on the Kptfile repository name  for which packages to update")
 	cmd.Flags().StringVarP(&o.KptBinary, "bin", "", "", "the 'kpt' binary name to use. If not specified this command will download the jx binary plugin into ~/.jx3/plugins/bin and use that")
 	cmd.Flags().StringVarP(&o.Strategy, "strategy", "s", "resource-merge", "the 'kpt' strategy to use. To see available strategies type 'kpt pkg update --help'. Typical values are: resource-merge, fast-forward, force-delete-replace")
+	cmd.Flags().StringVarP(&o.ContainerTool, "container-tool", "c", defaultContainer, "the underlying container tool for kpt to use, default is "+defaultContainer)
```

Comment:
You don't need to explicitly put the default value into the usage, it will be shown in the help automatically